test/test_plugins.py

```diff
@@ -123,3 +126,34 @@ def test_plugin_load_pypackage_bad_dir_no_init(tmpdir):
 
     with pytest.raises(Exception):
         plugins.handlers.PyFilePlugin(package_dir.strpath)
+
+
+def test_plugin_load_entrypoint(tmpdir):
+    root = tmpdir.mkdir('loader_mods')
+    mod_file = root.join('file_mod.py')
+    mod_file.write(MOCK_MODULE_CONTENT)
+
+    # generate setuptools Distribution object
+    distrib = pkg_resources.Distribution(root.strpath)
+    sys.path.append(root.strpath)
+
+    # load the entrypoint
```

Comment:
The one place in your docstrings/comments I found that uses "entrypoint" as one word instead of two! 😁

```suggestion
    # load the entry point
```